src/libc/internal/aarch64.rs

```diff
@@ -34,7 +32,10 @@ unsafe extern "C" fn __steed_clone() {
     //
     // We save `fn_` and `arg` on the child stack.
 
-    asm!("
+    global_asm!("
```

Comment:
you could even commit actual .s files, if you're interested, and do `global_asm!(include_str!("./asm_file.s"));`